src/libs/KeyboardShortcut/index.js

```diff
@@ -124,6 +126,56 @@ function subscribe(key, callback, modifiers = 'shift', captureOnInputs = false)
     return () => unsubscribe(key);
 }
 
+
+/**
+ * Captures all the key inputs until explictly consumed.
+ *
+ * @param {boolean} [captureOnInputs=false] Should we capture the event on inputs too?
+ */
+function buffer(captureOnInputs = false) {
+    if (keyBufferCallback) {
+        return;
+    }
+    keyBufferCallback = (evt) => {
+        if (!captureOnInputs
+            && evt.target.nodeName === 'INPUT'
+            && evt.target.nodeName === 'TEXTAREA'
+            && evt.target.contentEditable === 'true') {
+            return;
+        }
```

Comment:
Does this mean we are going to early return if the user is typing in an input box normally?

Reply:
Yeah. This is supposed to work in that way. If the user is typing in the focused input then don't buffer the value. Useful in cases where we want to use the buffer in an input that will be autofocused at some moment in time. So to minimize duplicate